### ISSUE TITLE
ipod: rolecard is breaking on smallest device size. 

### ISSUE THREAD

**Issue by jpailthorpe** [engagementgamelab/at-stake-v3#156]
This has to do with mockups not taking into account additional screen real estate by browser. currently on devices of this size, you cannot see your hidden agenda items. 

![img_0141](https://cloud.githubusercontent.com/assets/10051544/24985133/331887a0-1fc0-11e7-82da-49fdbdad693e.JPG)

**Comment by jpailthorpe** [comment]
I need to do a redesign of the role card to include the small device sizes. putting a pending on this.

**Comment by jpailthorpe** [comment]
redesign finished. Another issue has this redesign in it, so moving to done.
